common4j/src/main/com/microsoft/identity/common/java/eststelemetry/SchemaConstants.java

```diff
@@ -54,6 +56,17 @@ public static final class Key {
         public static final String FRT_STATUS = TelemetryEventStrings.Key.FRT_STATUS;
         public static final String MRRT_STATUS = TelemetryEventStrings.Key.MRRT_STATUS;
         public static final String ALL_TELEMETRY_DATA_SENT = "is_all_telemetry_data_sent";
+
+        // flw and multiple reg fields
+        public static final String IS_SHARED_DEVICE = "isSharedScenario";
```

Comment:
isSharedDevice

Reply:
I think it is called `isSharedScenario` in the schema.

Reply:
Can we get it changed in the schema as well, or that is locked? As IMO isSharedDevice is clearer than isSharedScenario.

Reply:
@iamgusain You can suggest that in the design doc to Olga. It honestly doesn't matter what the key is. (Keys are not emitted in eSTS telemetry)